### PR TITLE
Add grid fallback to story promo

### LINKED ISSUE #471: Add grid fallback to story promo
**Is your feature request related to a problem? Please describe.**
Now that we have a [story promo component](https://github.com/bbc/psammead/tree/latest/packages/components/psammead-story-promo) we need to add support for a grid fallback.

**Describe the solution you'd like**
Add a fallback for grid that is supported for ie9,10&11

This can use max widths, as seen in the articles layout grid items [such as this example](https://github.com/bbc/simorgh/blob/latest/src/app/lib/layoutGrid.js#L111-L132)

Note, we have moved away from using ms-grid in articles due to its lack of IE9 support and issues with rows. While the rows issue wont apply here, using max-width should be sufficient. Info about this move away from ms-grid can be found in https://github.com/bbc/simorgh/pull/1376

**Testing notes**
Dev insight: Visual, manual device testing and snapshots

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #471

**Overall change:**
Add a grid fallback to the story promo, that is supported for ie9,10&11.

**Code changes:**
- Use `max-widths` when `grid` is not supported
- Add `grid` styles within `@supports` CSS at-rule
- Replace sample `Image`
---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by dr3 — CHANGES_REQUESTED** [review]
Tested on IE11 & IE10, looks good.
Tested on IE9 & it doesnt seem to work.

Testing steps:
- In simorgh `npm i @bbc/psammead-story-promo`
- Change ArticleMain/index.js to [this code](https://gist.github.com/dr3/3c969d35b1e64b580cce004c9a15c08d)
- Run `npm link ../psammead/packages/components/psammead-story-promo`
- Run `npm run build` in psammead
- Run `npm run build && npm run start` in simorgh
- Open in browserstack on IE9
- See the issue
image

**Comment by david-boydell** [comment]
Hi Dennis, I've found that when viewed as RTL the padding to separate the image and the text is not present.

image

**Comment by DenisHdz** [comment]
> Tested on IE11 & IE10, looks good.
> Tested on IE9 & it doesnt seem to work.
> 
> Testing steps:
> 
> * In simorgh `npm i @bbc/psammead-story-promo`
> * Change ArticleMain/index.js to [this code](https://gist.github.com/dr3/3c969d35b1e64b580cce004c9a15c08d)
> * Run `npm link ../psammead/packages/components/psammead-story-promo`
> * Run `npm run build` in psammead
> * Run `npm run build && npm run start` in simorgh
> * Open in browserstack on IE9
> * See the issue
>   image

Tested on IE9 and it works fine :)

There is an issue linking `psammead` to `simorgh`. A hack to fix this is to 
```cp ../psammead/packages/components/psammead-story-promo/dist/index.js ./node_modules/@bbc/psammead-story-promo/dist/index.js```

Screenshot 2019-05-08 at 15 39 39

**Review by j-pendlebury — DISMISSED** [review]
looks good to me, just need to resolve some conflicts ⬇️

File: packages/components/psammead-story-promo/src/index.jsx [hunk 1]
```diff
@@ -19,33 +20,78 @@ import {
...
 
 const TextGridItem = styled.div`
-  grid-column: 3 / span 4;
+  display: inline-block;
+  vertical-align: top;
+  padding-left: ${GEL_SPACING};
+  max-width: ${fourOfSixColumnsMaxWidthScaleable};
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    padding-left: ${GEL_SPACING_DBL};
```
Comment (dr3) [inline packages/components/psammead-story-promo/src/index.jsx]: Ideally these would be consolidated into `padding: 0 ${GEL_SPACING_DBL}`

**Review by dr3 — CHANGES_REQUESTED** [review]
image
At smaller breakpoints this doesnt seem to work for RTL

**Review by dr3 — APPROVED** [review]
Great work :D

**Review by j-pendlebury — APPROVED** [review]
LGTM 👍

**Comment by david-boydell** [comment]
Looking good across all IE browsers

**Pull request merged**